modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/api/plugin/DuplicateExportedPackageException.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.runtime.module.artifact.activation.api.plugin;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Thrown to indicate that more than one artifact is exporting a given package.
+ * 
+ * @since 4.5
+ */
+public class DuplicateExportedPackageException extends PluginResolutionError {
```

Comment:
where is this used?

Reply:
It's used in `BundlePluginDependenciesResolver`.